docker/pom.xml

```diff
@@ -38,55 +38,65 @@
         <concurrent-api.version>2.0.0</concurrent-api.version>
         <!-- Jakarta Interceptors -->
-        <jakarta.interceptor-api.version>2.0.0</jakarta.interceptor-api.version>
+        <jakarta.interceptor-api.version>2.1.0-RC1</jakarta.interceptor-api.version>
         <!-- Jakarta EE Security + Authentication/Authorization -->
         <jakarta.security.enterprise-api.version>2.0.0</jakarta.security.enterprise-api.version>
-        <jakarta.authorization-api.version>2.0.0</jakarta.authorization-api.version>
+        <jakarta.authorization-api.version>2.1.0-RC1</jakarta.authorization-api.version>
         <jakarta.authentication-api.version>2.0.0</jakarta.authentication-api.version>
         <!-- Jakarta Messaging -->
         <jms-api.version>3.0.0</jms-api.version>
-        <jakarta.activation-api.version>2.1.0-RC2</jakarta.activation-api.version>
+        <jakarta.activation-api.version>2.1.0</jakarta.activation-api.version>
         <!-- Jakarta Persistence -->
         <jakarta-persistence-api.version>3.1.0-RC1</jakarta-persistence-api.version>
         <!-- Jakarta Transactions -->
         <jakarta.transaction-api.version>2.0.0</jakarta.transaction-api.version>
         <!-- Jakarta Connectors -->
         <jakarta.resource-api.version>2.0.0</jakarta.resource-api.version>
         <!-- Jakarta Batch -->
-        <jakarta.batch-api.version>2.0.0</jakarta.batch-api.version>
+        <jakarta.batch-api.version>2.1.0-M1</jakarta.batch-api.version>
         <!-- Jakarta Enterprise beans -->
         <jakarta.ejb-api.version>4.0.0</jakarta.ejb-api.version>
         <!-- Jakarta JSON -->
         <jsonp-api.version>2.1.0</jsonp-api.version>
         <json.bind-api.version>3.0.0-SNAPSHOT</json.bind-api.version>
         <!-- Jakarta Server Pages -->
-        <jsp-api.version>3.0.0</jsp-api.version>
+        <jsp-api.version>3.1.0</jsp-api.version>
         <!-- Jakarta Standard Tag Library -->
         <jstl-api.version>2.0.0</jstl-api.version>        
         <!-- CDI -->
         <cdi-api.version>3.0.0</cdi-api.version>
         
         <jakarta.annotation-api.version>2.1.0</jakarta.annotation-api.version>
         <jakarta.el-api.version>5.0.0</jakarta.el-api.version>
-        <jakarta.inject-api.version>2.0.0</jakarta.inject-api.version>
+        <jakarta.inject-api.version>2.0.1</jakarta.inject-api.version>
         <jakarta.mail-api.version>2.1.0</jakarta.mail-api.version>
         <jakarta.servlet-api.version>6.0.0</jakarta.servlet-api.version>
-        <jakarta.validation-api.version>3.0.0</jakarta.validation-api.version>
+        <jakarta.validation-api.version>3.0.2-SNAPSHOT</jakarta.validation-api.version>
 
-        <jakarta.ws.rs-api.version>3.0.0</jakarta.ws.rs-api.version>
-        <jakarta.xml.bind-api.version>4.0.0-RC2</jakarta.xml.bind-api.version>
+        <jakarta.ws.rs-api.version>3.1.0</jakarta.ws.rs-api.version>
+        <jakarta.xml.bind-api.version>4.0.1-SNAPSHOT</jakarta.xml.bind-api.version>
```

Comment:
Where is the jakarta.xml.bind-api 4.0.1-SNAPSHOT staged?

Reply:
I think 4.0.1-SNAPSHOT is a mistake. That's the version automatically set in the API project, but the latest release was 4.0.0-RC3 with *maybe* a 4.0.0-RC4 coming up still.

Reply:
@scottmarlow - https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/xml/bind/jakarta.xml.bind-api/4.0.1-SNAPSHOT/ is the latest.

Reply:
@gurunrao I was able to resolve the snapshot artifact via the staging repo (confirmed by running `docker/build_signatures.sh` locally with your branch).

@lukasj with this change, we will temporarily use the XML Binding snapshot build when generating TCK signature map files that depend on XML Binding (e.g. jakarta.xml.ws.sig_3.0_se11 + jakarta.ws.rs.sig_3.1_se11 currently).  

Do you expect to release a XML Binding 4.0.0-RC4 artifact soon?

Reply:
why 4.0.0-RC3 cannot be used?

Reply:
no reason that I know of.  Thanks for asking.  

@gurunrao could you please switch to `4.0.0-RC3`.

Reply:
I will switch 4.0.0-RC3, it will be good to delete https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/xml/bind/jakarta.xml.bind-api/4.0.1-SNAPSHOT/.

Reply:
there is no way and no reason to delete the SNAPSHOT. It's a build built by every commit and there were changes between RC3 and today (these changes do not affect API signatures, AFAICT)